benchmarks/dynamo/huggingface_models_list.txt

```diff
@@ -1,53 +1,51 @@
 AlbertForMaskedLM,8
 AlbertForQuestionAnswering,8
-AllenaiLongformerBase,1
-BartForCausalLM,16
+AllenaiLongformerBase,8
```

Comment:
so sizes in this file are maximum size for eager?

Reply:
Yes